### PR TITLE
Add changelog for 5.0.0-alpha3

### PR DESCRIPTION
**Pull request by suyograo**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,40 @@
...
+   able to start without this file, so please make sure to pass in `path.settings` if you are starting Logstash manually after 
+   installing it via a package (RPM, DEB) ([#4401](https://github.com/elastic/logstash/issues/4401)).
+ - Breaking Change: Most of the long form options (https://www.elastic.co/guide/en/logstash/5.0/command-line-flags.html) 
+   have been renamed to adhere to the yml dot notation to be used in the settings file. Short form options have not been
+   changed ([#4401](https://github.com/elastic/logstash/issues/4401)).
+ - Breaking Change: When Logstash is installed via DEB, RPM packages, it uses /usr/share and /var to install binaries and 
+   config files respectively. Previously it used to install in /opt directory. This change was done to make the user experience 
+   consistent with other Elastic products ([#5101](https://github.com/elastic/logstash/issues/5101)).
+ - Breaking Change: For plugin developers, the Event class has a [new API](https://github.com/elastic/logstash/issues/5141) 
+   to access its data. You will no longer be able to directly use the Event class through the ruby hash paradigm.
```
Comment (jsvd) [inline CHANGELOG.md]: maybe worth mentioning all default plugins and been refactored and major bumped due to this?

**Comment by jsvd** [comment]
I'm not super familiar with all the latest changes in default plugins but change LGTM, other than my small comment.

File: CHANGELOG.md (hunk 1 above)
Comment (suyograo) [inline CHANGELOG.md]: +1

**Comment by suyograo** [comment]
@jsvd added note about plugins updated for Java Event.

**Comment by suyograo** [comment]
Merging.

**Pull request merged**